libvast/test/format/mrt.cpp

```diff
@@ -57,9 +57,10 @@ TEST(MRT) {
   /// - # state changes: grep STATE_CHANGE plain.txt | grep -v "Subtype" | wc
   /// - # withdrawals: grep "Withdrawn Routes: " plain.txt | wc
   /// - # announcements: grep "NLRI: " plain.txt | wc
-  CHECK_EQUAL(events["mrt.bgp4mp.state_change"], 46u);
-  CHECK_EQUAL(events["mrt.bgp4mp.update.withdrawn"], 2105);
-  CHECK_EQUAL(events["mrt.bgp4mp.update.announcement"], 24328u);
+  // FIXME These tests are horribly broken.
```

Comment:
Are the tests broken, or the functionality that they test? In the former case, why not just remove them instead of commenting out?

Either way, should we link to the clubhouse story here to add more information?

Reply:
We don't usually link to ClubHouse in public repos, but I've added a small label in these two cases for future reference as an exception, as these really are bugs that we need to fix.

Reply:
Interesting, it would be cool if clubhouse had a feature to make certain stories public, or to mirror some stories with github issues. In general it would seem to be useful to be able to link somewhere with further context, investigation, etc.